notebooks/workflows/adviser-inspection-workflow.ipynb

```diff
@@ -0,0 +1,254 @@
+{
```

Comment:
<p><code>json</code></p><p><code>&nbsp;&nbsp;"THOTH_ADVISER_LIBRARY_USAGE": {</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;"report": {},</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;"version": "string"</code></p><p><code>&nbsp;&nbsp;},</code></p><p> </p><p> </p><p>should be string, not an object</p><p> </p><p><code>json</code></p><p><code>"THOTH_ADVISER_LIBRARY_USAGE": "{ \"report\": {}, \"version\": \"string\" }",</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thoth-station/notebooks/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>